src/components/UserCategoryNav.jsx

```diff
@@ -0,0 +1,46 @@
+import {GridFour, CookingPot, NotePencil, Brain} from "@phosphor-icons/react";
+
+const UserCategoryNav = () => {
+  return (
+    <div className="flex gap-8">
+      <div className="flex  flex-col ">
+        <div className="bg-black w-full h-1 "></div>
+        <div className="flex flex-row gap-2 text-xxs font-semibold items-center mt-2 w-3/4 px-2">
+          <div>
```

Comment:
Is this extra div required?

Reply:
Yes. It's not functioning right now. but when the link is selected it should have a black bar above it